src/tiled/stampbrush.cpp

```diff
@@ -242,6 +243,18 @@ void StampBrush::populateToolBar(QToolBar *toolBar)
     mStampActions->populateToolBar(toolBar, mIsRandom, mIsWangFill);
 }
 
+void StampBrush::setWangSet(WangSet *wangSet)
+{
+    mWangFiller->setWangSet(wangSet);
+
+    mMissingTilesets.clear();
+
+    const SharedTileset &tileset = wangSet->tileset()->sharedPointer();
```

Comment:
This is crashing when the `wangSet` is null (happens to me when I create a new map, for example).